Modules/_winapi.c

```diff
@@ -325,6 +325,7 @@ static PyType_Slot winapi_overlapped_type_slots[] = {
     {Py_tp_doc, "OVERLAPPED structure wrapper"},
     {Py_tp_methods, overlapped_methods},
     {Py_tp_members, overlapped_members},
+    {Py_tp_new, _PyType_DisabledNew},
```

Comment:
"'_PyType_DisabledNew': undeclared identifier [D:\a\cpython\cpython\PCbuild\pythoncore.vcxproj]" says the CI. Maybe you forgot an include?

Reply:
Let's wait with this until we resolve https://github.com/python/cpython/pull/25653#discussion_r622326954. 856f8247687703e0e66c3db83035d20fe3c36fc6 built just fine :)

Reply:
Doh, yes, I forgot the include. Thanks!

Reply:
Hm, Windows builds are still broken.

Reply:
Of course, the Windows build does not use setup.py nor Modules/Setup. So I guess I need to modify something inside PCbuild to get this working. I'll look at it later.